sale_force_invoiced/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===================
+Sale Force Invoiced
+===================
+
+This module adds the possibility for users to force the invoice status of the
+sales orders to 'Invoiced', even when not all the quantities ordered or
+delivered have been invoiced.
+
+This feature useful in the following scenario:
+
+* The customer disputes the quantities to be invoiced for, after the
+  products have been delivered to her/him, and you agree to reduce the
+  quantity to invoice (without sending a refund).
+
+* When migrating from a previous Odoo version, in some cases there is less
+  quantity invoiced to what was delivered, and you don't want these old sales
+  orders to appear in your 'To Invoice' list.
+
+
+Usage
+=====
+
+#. Create a sales order and confirm it.
+#. Deliver the products/services.
+#. Create an invoice and reduce the invoiced quantity. The sales order
+   invoicing status is 'To Invoice'.
+#. Change the status of the sales order to 'Done'.
```

Comment:
Also, I would like you to modify this line to something like this:

Lock the Sale Order, to change the status of it to 'Done'.

@mreficent what do you th

Reply:
ok